packages/node/src/evm/fulfillments/api-calls.ts

```diff
@@ -53,11 +46,11 @@ async function testFulfill(
     `Attempting to fulfill API call with status code:${statusCode.toString()} for Request:${request.id}...`
   );
 
-  const operation = () =>
+  const operation = (): Promise<StaticResponse> =>
     airnodeRrp.callStatic.fulfill(
       request.id,
-      // TODO: make sure airnodeId is not null
-      request.airnodeId!,
+      // TODO: make sure airnodeAddress is not null
```

Comment:
I wonder if we still need to check what the TODO says 🤔

Reply:
I think at that point it must be defined, but TS type is still `string | null`. We could maybe rethink the typings to avoid `!`, but that doesn't have a high prio.